### ISSUE TITLE
How do I ignore an instance of TooWordy?

### ISSUE THREAD

**Issue by iainelder** [errata-ai/write-good#10]
I've written "Set the minimum version for each dev dependency."

Vale warns that "minimum" is too wordy.

In this technical context I think it's the right word to use.

Can I tell Vale to ignore just this instance of the word (say by line number) and warn me anywhere else?

**Comment by jdkato** [comment]
You can disable the rule by adding [comments](https://vale.sh/docs/topics/config/#markup-based-configuration) around the specific line.

**Comment by iainelder** [comment]
I had hoped there was a less "wordy" solution to the problem, but that's for clarifying that this is the supported way.
